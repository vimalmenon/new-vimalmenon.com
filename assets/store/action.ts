import * as session from "./session/session.action";
import * as config from "./config/config.action";
import * as admin from "./admin/admin.action";

export {
	session,
	config,
	admin
};