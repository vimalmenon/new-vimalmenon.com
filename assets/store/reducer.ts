import { combineReducers } from "redux";

import session from "./session/session.reducer";
import config from "./config/config.reducer";

const reducer = combineReducers<IState>({
	session,
	config
});

export default reducer;