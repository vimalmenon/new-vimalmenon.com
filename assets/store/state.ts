import session from "./session/session.state";
import config from "./config/config.state";
import admin from "./admin/admin.state";

const state:IState = {
	session,
	config,
	admin
};

export default state;