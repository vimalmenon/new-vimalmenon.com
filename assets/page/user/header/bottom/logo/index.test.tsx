import "jsdom-global/register";
import React from "react";
import renderer from "react-test-renderer";
import FooterBottom from "./index";
import {mount, shallow} from "enzyme";

import { Provider } from "react-redux";
import store from "store";
import {
	BrowserRouter as Router
} from "react-router-dom";

describe("SocialMedia Component", ()=> {
    test("Snapshot Testing", () => {
        const component = renderer.create(<Provider store={store}><Router><FooterBottom /></Router></Provider>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});