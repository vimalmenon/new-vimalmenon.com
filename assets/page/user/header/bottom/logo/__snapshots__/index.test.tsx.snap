// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialMedia Component Snapshot Testing 1`] = `
<div
  className="makeStyles-root-1"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Vimal Menon"
      height="100%"
      src="test-file-stub"
      width="100%"
    />
  </a>
</div>
`;
